frontend/express/public/core/events/javascripts/countly.overview.models.js

```diff
@@ -195,21 +351,44 @@
                             context.commit("setMonitorEvents", res || {});
                             var events = [];
                             if (res && res.overview) {
+                                context.commit("setConfigureOverview", res.overview.slice());
                                 for (var i = 0; i < res.overview.length; i++) {
                                     events.push(res.overview[i].eventKey);
                                 }
                             }
-                            countlyEventsOverview.service.fetchMonitorEventsData(events, context)
-                                .then(function(response) {
-                                    if (response) {
-                                        return context.commit("setMonitorEventsData", countlyEventsOverview.helpers.getMonitorEvents(response, context) || []);
+                            countlyEventsOverview.service.fetchGroupData(context)
+                                .then(function(result) {
+                                    if (result) {
+                                        context.commit("setOverviewGroupData", result);
+                                        context.commit("setConfigureEventsList", countlyEventsOverview.helpers.getOverviewConfigureList(res, result));
+                                        context.commit("setEventMapping", countlyEventsOverview.helpers.getEventMapping(res, result));
+                                        countlyEventsOverview.service.fetchMonitorEventsData(events, context)
+                                            .then(function(response) {
+                                                if (response) {
+                                                    return context.commit("setMonitorEventsData", countlyEventsOverview.helpers.getMonitorEvents(response, context) || []);
+                                                }
+                                            });
                                     }
                                 });
                         }
                     });
             },
             fetchSelectedDatePeriod: function(context, period) {
                 context.commit('setSelectedDatePeriod', period);
+            },
+            fetchEventProperties: function(context, selectedEvent) {
```

Comment:
**fetchEventProperties** is comming **setEventProperties**  😄 
naming could have been better. 
just a note for future.

Reply:
Sure :)